orca-webhook/src/main/java/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.java

```diff
@@ -0,0 +1,52 @@
+package com.netflix.spinnaker.orca.webhook.tasks;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.spinnaker.orca.api.pipeline.RetryableTask;
+import com.netflix.spinnaker.orca.api.pipeline.TaskResult;
+import com.netflix.spinnaker.orca.api.pipeline.models.StageExecution;
+import com.netflix.spinnaker.orca.webhook.config.WebhookProperties;
+import com.netflix.spinnaker.orca.webhook.service.WebhookService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+public class CreateWebhookTask implements RetryableTask {
+  private long backoffPeriod = 10000;
```

Comment:
these could change to static final - I think in groovy they were this way because the became properties that implemented the RetryableTask interface

Reply:
moved into the getters